src/http/ngx_http_mruby_module.c

```diff
@@ -136,6 +136,11 @@ static ngx_int_t ngx_http_mruby_content_inline_handler(ngx_http_request_t *r);
 static ngx_int_t ngx_http_mruby_log_inline_handler(ngx_http_request_t *r);
 
 #if (NGX_HTTP_SSL) && OPENSSL_VERSION_NUMBER >= 0x1000205fL
+ngx_connection_t *ngx_mruby_connection = NULL;
+ngx_connection_t *ngx_mrb_get_connection(void)
+{
+  return ngx_mruby_connection;
+}
```

Comment:
You don't need to use global variable `ngx_connection_t` if `ngx_http_mruby_srv_conf_t *mscf` have `ngx_connection_t ` member like `mscf->c` and get `ngx_http_mruby_srv_conf_t *mscf = mrb->ud; ngx_connection_t *c =  mscf->c` in method functions.